### PR TITLE
added utility to wrap long messages with provided limit

### PR DESCRIPTION
**Pull request by Kathuria**
#62 With this PR, suggesting an option to wrap very long messages with provided limit.
Thus, created a utility for now and if required can be consumed with a limit that can help for most use cases.

### CODE REVIEW

**Comment by AutomationPanda** [comment]
Thanks for your pull request, @Kathuria! I added the label `hacktoberfest-accepted` so that it will count towards your Hacktoberfest goal. I'll review your changes within the next week. In the meantime, it looks like your branch is out-of-date with the main branch, so please update your branch.

**Comment by AutomationPanda** [comment]
@Kathuria , please also update the changelog:
https://q2ebanking.github.io/boa-constrictor/contributing/contributing-code/#7-making-code-changes

**Comment by Kathuria** [comment]
Thanks @AutomationPanda : Updated changes as suggested.

File: Boa.Constrictor/Utilities/MessagesWrapper.cs [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    /// </summary>
+    public static class MessagesWrapper
+    {
+        /// <summary>
+        /// Wraps long lines of messages with provided limit of text to display as wrapped line.
+        /// </summary>
+        /// <param name="message">The Message on which word wrap will apply.</param>
+        /// <param name="limit">The limit of characters to be applied.</param>
+        /// <returns></returns>
+        public static List<string> GetWrappedMessage(string text, int limit)
```
Comment (AutomationPanda) [inline Boa.Constrictor/Utilities/MessagesWrapper.cs:16]: You must add a `using` statement to use `List<string>`.

Here's the error message: 
```
D:\a\boa-constrictor\boa-constrictor\Boa.Constrictor\Utilities\MessagesWrapper.cs(16,23): error CS0246: The type or namespace name 'List<>' could not be found (are you missing a using directive or an assembly reference?) [D:\a\boa-constrictor\boa-constrictor\Boa.Constrictor\Boa.Constrictor.csproj]
```

**Review by AutomationPanda — CHANGES_REQUESTED** [review]

File: Boa.Constrictor/Utilities/MessagesWrapper.cs (hunk 1 above)
Comment (Kathuria) [reply Boa.Constrictor/Utilities/MessagesWrapper.cs:16]: @AutomationPanda : let me fix this.

Comment (Kathuria) [reply Boa.Constrictor/Utilities/MessagesWrapper.cs:17]: @AutomationPanda : Sorry for the delay, this should be fixed now.

**Review by AutomationPanda — APPROVED** [review]
If you get a chance, please add unit tests.

**Pull request merged**